Dockerfile.dyninst

```diff
@@ -0,0 +1,58 @@
+# Build stage with Spack pre-installed and ready to be used
+# docker build -f Dockerfile.dyninst -t ghcr.io/autamus/dyninst:05-25-master .
+FROM spack/ubuntu-bionic:latest as builder
+
+RUN git clone https://github.com/autamus/registry && \
```

Comment:
I'm assuming autamus keeps a regular build of the HEAD of master for Dyninst?

Reply:
it does not - it only builds releases. We need this Dockerfile so we can get a build for the base (with your fixes).